dmoj/contrib/testlib.py

```diff
@@ -14,7 +14,7 @@ class ContribModule(DefaultContribModule):
     PARTIAL = 7
 
     name = 'testlib'
-    repartial = re.compile(br'^points (\d+)\n$')
+    repartial = re.compile(br'^points (\d+)\n$', re.M)
```

Comment:
Do we need/wait the `\n`?

Reply:
IMO we should maintain parity with testlib.h, whatever that may be...

Also I would recommend against removing entirely, as then we'd have to modify every checker that uses partials...

Reply:
No I mean the regex will match the same either way. Do we just want to not care if testlib outputs the trailling `\n` or not?

Reply:
Decided to drop the trailing `\n`.